content/en/docs/pipelines/sdk/pipelines-metrics.md

```diff
@@ -21,30 +21,42 @@ pipeline agent uploads the local file as your run-time metrics. You can view the
 uploaded metrics as a visualization in the **Runs** page for a particular
 experiment in the Kubeflow Pipelines UI.
  
-## Export the metrics file
+## Export the metrics dictionary
 
-To enable metrics, your program must write out a file named 
-`/mlpipeline-metrics.json`. For example:
+To enable metrics, your component must have an output called "MLPipeline Metrics".
+JSON-serialized metrics dictionary should be returned from that output.
```

Comment:
Suggestion:

- Refactor (from passive to active voice) "JSON-serialized metrics dictionary should be returned from that output":

```suggestion
Then, that output should return a JSON-serialized metrics dictionary as follows:
```

Reply:
Done.